core/services/privacy/export/PersonalDataExporterInterface.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+namespace EventEspresso\core\services\privacy\export;
+
+defined('EVENT_ESPRESSO_VERSION') || exit('No direct script access allowed');
+
```

Comment:
same here, no side effects in files that defines symbols

Reply:
ya